### PR TITLE
Naraven/rhel compliant

### PR DESCRIPTION
**Pull request by NaraVen**
1) Made command line args RHEL compliant"
2) user agent blobfuse
3) removed validation to allow any order in config file
4) made version number -beta

### CODE REVIEW

File: cpplite/include/constants.dat [hunk 1]
```diff
@@ -116,7 +116,7 @@ DAT(header_value_payload_format_nometadata, "application/json;odata=nometadata")
 DAT(header_value_payload_format_fullmetadata, "application/json;odata=fullmetadata")
 DAT(header_value_storage_blob_version, "2018-11-09")
 
-DAT(header_value_user_agent, "azure-storage-cpplite/0.3.0")
```
Comment (vibhansa-msft) [inline cpplite/include/constants.dat:119]: There is no need to change this. We have taken this from the cpplite lib itself.

File: blobfuse/src/BlobfuseGlobals.cpp [hunk 2]
```diff
@@ -29,22 +29,27 @@ AUTH_TYPE get_auth_type(std::string authStr)
 {
     if(!authStr.empty()) {
         std::string lcAuthType = to_lower(authStr);
-        if (lcAuthType == "msi") {
+        syslog(LOG_DEBUG, "Auth type set as %s", lcAuthType.c_str());
+        int lcAuthTypeSize = (int)lcAuthType.size();
+         // sometimes an extra space or tab sticks to authtype thats why this size comparison, it is not always 3 lettered
+        if(lcAuthTypeSize > 0 && lcAuthTypeSize < 5)
```
Comment (vibhansa-msft) [inline blobfuse/src/BlobfuseGlobals.cpp:35]: Even if do the size check to safeguard extra space and tab, below we are comparing the data without handling the same. We should rather trim the value first and then do the comparisons.

File: cpplite/include/constants.dat (hunk 1 above)
Comment (NaraVen) [reply cpplite/include/constants.dat:119]: No it should show Azure-storage-fuse and not cpplite. The logs are triaged based on Azure-storage-fuse. I understand it is not correct to change the Cpplite code base but the endproduct wins here. So the logs have to show Azure-storage-fuse. Leaving this as is

File: blobfuse/src/BlobfuseGlobals.cpp (hunk 2 above)
Comment (NaraVen) [reply blobfuse/src/BlobfuseGlobals.cpp:35]: done trimmed.

**Pull request merged**